### PR TITLE
Added outdated warning to all pipelines v1 pages

### PR DESCRIPTION
**Pull request by hbelmiro**
Resolves: https://github.com/kubeflow/website/issues/3751

### CODE REVIEW

**Comment by hbelmiro** [comment]
cc @chensun @thesuperzapper

**Comment by thesuperzapper** [comment]
@hbelmiro looks good, simple fix.

/lgtm

**Comment by thesuperzapper** [comment]
@hbelmiro Although we may want to have a new "out of date" warning for all the pages in the "Legacy V1" section:

https://www.kubeflow.org/docs/components/pipelines/legacy-v1/

Which explains that users are looking at the docs for Kubeflow Pipelines 1.0, which is no longer supported (but which Kubeflow Pipelines 2.0 supports as backwards compatible).

**Comment by hbelmiro** [comment]
@thesuperzapper something like the following?

```
{{% alert title="Out of date" color="warning" %}}
This guide contains outdated information pertaining to Kubeflow v1. You may want to access the documentation for the latest Kubeflow Pipelines version that supports v1 as backward compatible.
{{% /alert %}}
```

cc @rimolive

**Comment by hbelmiro** [comment]
@thesuperzapper 

> @hbelmiro I think something more direct about telling users to see the new docs (and linking them).
> 
> ```
> {{% alert title="Old Version" color="warning" %}}
> This page is about __Kubeflow Pipelines V1__, for information about __Kubeflow Pipelines V2__, please see the [new docs](__root_link_to_v2_docs__).
> 
> Please note that Kubeflow Pipelines V2 supports running V1 pipelines in a [backwards compatible mode](__page_which_explains_this__).
> {{% /alert %}}
> ```

Added.



> On the page which explains the backwards compatibility we should outline:
> 
>     1. You need to use the KFP V1 SDK to submit/compile the pipelines to V2 (you can't use the V2 SDK for this)
> 
>     2. That the new object store integration is different.
> 
>     3. A brief overview of the differences between V1 / V2 (we already have a migration page, so perhaps just link this).
> 
>     4. A dicussion of how long we plan to keep backwards compatibility for V1 in KFP.

This is out of the scope of this PR. Can I ask you to open an issue? We can then add it to https://github.com/kubeflow/website/issues/3712.

**Comment by thesuperzapper** [comment]
@hbelmiro we can probably include most of the information people need to know in the warning itself, and just link them to the migration page.

How about we use this warning:

```
{{% alert title="Old Version" color="warning" %}}
This page is about __Kubeflow Pipelines V1__, please see the [V2 documentation](/docs/components/pipelines) for the latest information.

Note, while the V2 backend is able to run pipelines submitted by the V1 SDK, we strongly recommend [migrating to the V2 SDK](/docs/components/pipelines/user-guides/migration).
For reference, the final release of the V1 SDK was [`kfp==1.8.22`](https://pypi.org/project/kfp/1.8.22/), and its reference documentation is [available here](https://kubeflow-pipelines.readthedocs.io/en/1.8.22/).
{{% /alert %}}
```

Either way, I definitely think we can improve the V2 migration guide, because it's quite hard to follow right now, and does not really "sell you" on why you should go through the effort.

**Comment by rimolive** [comment]
@hbelmiro Open a follow-up issue with @thesuperzapper requests and let's move to other issues. There's much more important sections to cover in the new KFP docs.

**Comment by thesuperzapper** [comment]
@rimolive this PR does not block anything, so there is no need to rush it?

Also, I think it's very important for users to understand V1 vs V2 as right now its very confusing to users.

It a very straightforward update I have suggested in https://github.com/kubeflow/website/pull/3775#issuecomment-2195340281, and if @hbelmiro agrees, they can choose to update this PR so we can merge it.

**Comment by rimolive** [comment]
@thesuperzapper It does block the only contributor working on the remaining items for KFP docs, as listed in https://github.com/kubeflow/website/issues/3712.

Your suggestion can be worked later when we have the other pending topics done.

**Comment by thesuperzapper** [comment]
@hbelmiro I am disappointed this PR was merged without responding to my comments in my review https://github.com/kubeflow/website/pull/3775#issuecomment-2195340281.

The update I proposed was very important, so I have raised a new PR update the text:

- https://github.com/kubeflow/website/pull/3792

**Comment by rimolive** [comment]
Thanks for opening the issue. We'll be working to fix until 1.9 release.

**Comment by hbelmiro** [comment]
> @hbelmiro I am disappointed this PR was merged without responding to my comments in my review [#3775 (comment)](https://github.com/kubeflow/website/pull/3775#issuecomment-2195340281).

@thesuperzapper the issue that this PR addresses was originally to remove the warnings. You asked that [here](https://github.com/kubeflow/website/pull/3737#discussion_r1626480942), I created the issue, and sent this PR.
Then in this review, you asked to re-add the warning, and then change it, and then change it again, and again...

@rimolive left clear [here](https://github.com/kubeflow/website/pull/3775#issuecomment-2195535573) that we needed to move on with this PR to fix other pending issues.

**Comment by thesuperzapper** [comment]
@hbelmiro this warning is one of the most important parts of the docs, as there is a lot of confusion right now with V1 vs V2.

But in any case, if you agree with the update to the wording proposed in https://github.com/kubeflow/website/pull/3792, it should be ready to merge.
